deps/llvm.mk

```diff
@@ -73,17 +73,29 @@ LLVM_LIBCXX_TAR:=$(SRCDIR)/srccache/libcxx-$(LLVM_TAR_EXT)
 endif
 endif # LLVM_VER != svn
 
+# Figure out which targets to build (hardcode this to "host;NVPTX" when on 3.9+ only)
+LLVM_TARGETS := host
+ifeq ($(LLVM_VER_SHORT),$(filter $(LLVM_VER_SHORT),3.9 svn))
+LLVM_TARGETS += NVPTX
+endif
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+ifeq ($(LLVM_USE_CMAKE),1)
+LLVM_TARGET_STRING := $(subst $(SPACE),;,$(LLVM_TARGETS))
+else
+COMMA := ,
+LLVM_TARGET_STRING := $(subst $(SPACE),$(COMMA),$(LLVM_TARGETS))
```

Comment:
I figured I'd keep it a little more extensible, but sure that works too. I'll update the PR.